### ISSUE TITLE
VFS tools repo - accessibility package

### ISSUE THREAD

**Issue by oseasmoran73** [department-of-veterans-affairs/va.gov-team#30516]
## Description

- Determine why the accessibility package cannot be installed on linux

**Comment by timwright12** [comment]
closing as duplicate
